content/en/docs/security/otel-collector-configuration-best-practices.md

```diff
@@ -0,0 +1,139 @@
+---
+title: OpenTelemetry Collector Configuration Best Practices
+linkTitle: Collector config
+description:
+  Follow best practices to securely configure OpenTelemetry Collector.
+weight: 20
+---
+
+When setting up configuration for OpenTelemetry (OTel) Collector, consider the
+following practices to better secure your OTel Collector instance.
+
+<!--
+TODO: SHOULD only enable the minimum required components. What are those?
+--->
```

Comment:
This is a GREAT question. I think the minimum is 1 receiver and 1 exporter?

Reply:
👍 Included that as a statement in the ## Receivers and exporters section below.